external/java/src/test/java/ch/epfl/dedis/calypso/CalypsoTest.java

```diff
@@ -358,9 +358,16 @@ void multiLTS() throws CothorityException{
     }
 
     @Test
-    void reConnect() throws CothorityException {
+    void reConnect() throws CothorityException, InterruptedException, IOException {
         WriteInstance wr = doc.spawnWrite(calypso, publisherDarc.getBaseId(), publisher);
 
+        for (int i=0; i<3; i++){
```

Comment:
That looks like `git merge` instead of `git rebase`. Please use the latter for your PRs, it makes it easier to follow what you changed.